app/models/profile.rb

```diff
@@ -37,27 +37,39 @@ def gpo_verification_pending?
     gpo_verification_pending_at.present?
   end
 
+  def pending_reasons
+    [
+      *(:gpo_verification_pending if gpo_verification_pending?),
+      *(:fraud_check_pending if fraud_review_pending? || fraud_rejection?),
```

Comment:
`fraud_rejection` is meant to be when Fraud Ops has made a final determination that an account is fraudulent, so I don't think we should include it in `fraud_check_pending`.

If `fraud_rejection` is on a profile, it shouldn't be possible to activate it.

Whereas if `fraud_review_pending_at` has a value, we should set it to nil and activate the profile in a single transaction.